docs/modules/ROOT/pages/api/governance.adoc

```diff
@@ -10,11 +10,1946 @@
+* xref:#GovernorComponent-_cast_vote[`++_cast_vote(self, proposal_id, voter, support, reason, params)++`]
+--
+
+[.contract-index]
+.Events
+--
+* xref:#GovernorComponent-ProposalCreated[`++ProposalCreated(proposal_id, proposer, calls, signatures, vote_start, vote_end, description)++`]
+* xref:#GovernorComponent-ProposalQueued[`++ProposalQueued(proposal_id)++`]
+* xref:#GovernorComponent-ProposalExecuted[`++ProposalExecuted(proposal_id)++`]
+* xref:#GovernorComponent-ProposalCanceled[`++ProposalCanceled(proposal_id)++`]
+* xref:#GovernorComponent-VoteCast[`++VoteCast(voter, proposal_id, support, weight, reason)++`]
+* xref:#GovernorComponent-VoteCastWithParams[`++VoteCastWithParams(voter, proposal_id, support, weight, reason, params)++`]
+--
+
+[#GovernorComponent-Extensions-Traits]
+==== Extensions traits functions
+
+[.contract-item]
+[[GovernorComponent-GovernorSettingsTrait-voting_delay]]
+==== `[.contract-item-name]#++voting_delay++#++(self: @ContractState) → u64++` [.item-kind]#extension#
+
+Must return the delay, in number of timepoints, between the proposal is created and the vote starts. This can be
+increased to leave time for users to buy voting power, or delegate it, before the voting of a
+proposal starts.
+
+[.contract-item]
+[[GovernorComponent-GovernorSettingsTrait-voting_period]]
+==== `[.contract-item-name]#++voting_period++#++(self: @ContractState) → u64++` [.item-kind]#extension#
+
+Must return the delay, in number of timepoints, between the vote start and vote end.
+
+[.contract-item]
+[[GovernorComponent-GovernorSettingsTrait-proposal_threshold]]
+==== `[.contract-item-name]#++proposal_threshold++#++(self: @ContractState) → u256++` [.item-kind]#extension#
+
+Must return the minimum number of votes that an account must have to create a proposal.
+
+[.contract-item]
+[[GovernorComponent-GovernorQuorumTrait-quorum]]
+==== `[.contract-item-name]#++quorum++#++(self: @ContractState, timepoint: u64) → u256++` [.item-kind]#extension#
+
+Must return the minimum number of votes required for a proposal to succeed.
+
+[.contract-item]
+[[GovernorComponent-GovernorCountingTrait-counting_mode]]
+==== `[.contract-item-name]#++counting_mode++#++(self: @ContractState) → ByteArray++` [.item-kind]#extension#
+
+Must return a description of the possible `support` values for `cast_vote` and the way these votes are counted,
+meant to be consumed by UIs to show correct vote options and interpret the results. See `COUNTING_MODE()` for more details.
+
+[.contract-item]
+[[GovernorComponent-GovernorCountingTrait-count_vote]]
+==== `[.contract-item-name]#++count_vote++#++(ref self: ContractState, proposal_id: felt252, account: ContractAddress, support: u8, total_weight: u256, params: Span<felt252>) → u256++` [.item-kind]#extension#
+
+Must register a vote for `proposal_id` by `account` with a given `support`, voting `weight` and voting `params`.
+
```

Comment:
With this said, we should always backtick parameters in these descriptions with maybe this exception, I just don't think is worth checking and refactoring every description block to mention and backtick all the parameters.

Reply:
Fair points. If I find any instances where backticked params would without a doubt improve the description, I'll highlight them. Otherwise, keeping them as is works for me 👍